### DISCUSSION TITLE
[Feature] Generated Color Palettes

### DISCUSSION THREAD

**Discussion by shimizurei** [immich-app/immich#5395] [Feature Request]
### The feature

This has been my dream for a photo application. I thought I had found an app that had it, but when I went to look for it again, it was gone. Or it never existed. IDK. Can be opt-in. Not sure what would be a good color number limit (maybe 5 or 7?).

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
I don't think this is suited for core Immich, but if we ever get a chance to do plugin support in the future I think it'd be a great candidate for that!
